[2024-08-02T14:10:43.137+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-02T14:10:43.149+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.airbyte_check_sync_1 scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-08-02T14:10:43.157+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.airbyte_check_sync_1 scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-08-02T14:10:43.157+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-02T14:10:43.178+0700] {taskinstance.py:2330} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync_1> on 2024-08-01 00:00:00+00:00
[2024-08-02T14:10:43.181+0700] {standard_task_runner.py:64} INFO - Started process 11222 to run task
[2024-08-02T14:10:43.184+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline_supermarket_2', 'airbyte_check_sync_1', 'scheduled__2024-08-01T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_supermarket_2.py', '--cfg-path', '/tmp/tmpvtowuupf']
[2024-08-02T14:10:43.185+0700] {standard_task_runner.py:91} INFO - Job 112: Subtask airbyte_check_sync_1
[2024-08-02T14:10:43.223+0700] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_supermarket_2.airbyte_check_sync_1 scheduled__2024-08-01T00:00:00+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-02T14:10:43.301+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline_supermarket_2' AIRFLOW_CTX_TASK_ID='airbyte_check_sync_1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-01T00:00:00+00:00'
[2024-08-02T14:10:43.302+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-02T14:10:43.320+0700] {baseoperator.py:400} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-08-02T14:10:43.324+0700] {base.py:84} INFO - Using connection ID 'airflow-call-to-airbyte-example' for task execution.
[2024-08-02T14:10:43.343+0700] {airbyte.py:108} INFO - Job 78 completed successfully.
[2024-08-02T14:10:43.344+0700] {base.py:294} INFO - Success criteria met. Exiting.
[2024-08-02T14:10:43.344+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-02T14:10:43.348+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_supermarket_2, task_id=airbyte_check_sync_1, run_id=scheduled__2024-08-01T00:00:00+00:00, execution_date=20240801T000000, start_date=20240802T071043, end_date=20240802T071043
[2024-08-02T14:10:43.396+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-02T14:10:43.404+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-02T14:10:43.419+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-02T14:10:43.420+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
